app/src/gecko/java/org/mozilla/tv/firefox/ext/View.kt

```diff
@@ -26,6 +30,27 @@ fun View.validateKnownViewById(): Int {
     return this.id
 }
 
+/**
+ * Set the horizontal margins on the given view.
+ *
+ * We want to add padding to the beginning and end of the RecyclerView: ideally we'd just add
+ * paddingStart/End. Unfortunately, this causes a visual glitch as each card scrolls offscreen.
+ * Instead, we set the margins for the first and last card.
+ *
+ * See [DefaultChannelAdapter.onBindViewHolder] for an example
+ */
+fun View.setHorizontalMargins(context: Context, position: Int, itemCount: Int) {
```

Comment:
This functionality doesn't seem broadly applicable to the rest of the app.  I also feel that this is too narrow a use case for it to make sense polluting the View namespace.  Could we move this into a companion object for `ChannelTile`, or something similar?